src/events.rs

```diff
@@ -5,6 +5,7 @@ use std::time::Duration;
 #[derive(Debug)]
 pub enum Event {
     Error((UserRequest, HttpError)),
+    JoinError,
```

Comment:
I think the name can be confused with the join room action, may be something related to the user act abortion or task timing out?